src/preprocessing/passes/non_clausal_simp.cpp

```diff
@@ -16,32 +16,55 @@
 
 #include "preprocessing/passes/non_clausal_simp.h"
 
+#include <vector>
+
 #include "context/cdo.h"
 #include "options/proof_options.h"
 #include "smt/smt_statistics_registry.h"
 #include "theory/theory_model.h"
+#include "prop/registrar.h"
+#include "prop/cnf_stream.h"
+#include "prop/sat_solver_factory.h"
+#include "proof/proof_manager.h"
 
-#include <vector>
 
 using namespace CVC4;
 using namespace CVC4::theory;
 
 namespace CVC4 {
 namespace preprocessing {
+
+namespace {
+void handleConflictCase(AssertionPipeline* assertionsToPreprocess)
```

Comment:
I would actually inline this method again since it is now called from a single place and not doing anything complicated but this is a matter of taste to a certain extent.